### PR TITLE
Update ug delete

### PR DESCRIPTION
**Pull request by zzzlou**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -205,10 +207,14 @@ Examples: <br>
 
 Deletes the specified person from the address book.
 
-Format: `delete INDEX`
+Format: `delete INDICES`
 
-* Deletes the person at the specified `INDEX`.
+* Deletes the person(s) at the specified `INDEX` or range of indices.
```
Comment (seah-minlong) [inline docs/UserGuide.md:212]: at the specified `INDICES` could be better

File: docs/UserGuide.md [hunk 2]
```diff
@@ -205,10 +207,14 @@ Examples: <br>
...
 * The index refers to the index number shown in the **last displayed person list**.
+* `INDICES` can be a single number (e.g., `2`) or a closed range (e.g., `5-9`), separated by spaces (e.g., `1 2 3 5-9`).
+  * **Note:** In a closed range, there must be no spaces between the numbers and the hyphen (e.g., `5-9` is correct, 
+    but `5 - 9` or `5 -9` or `5- 9` is invalid).
+* `INDICES` should not be empty; at least one `INDEX` or `range` must be provided.
```
Comment (seah-minlong) [inline docs/UserGuide.md:217]: I don't think `INDEX` or `range` should be formatted as such as there is no other mention of it. The closest would be the "single number" and "closed range" above.

I would think writing `INDICES` should not be empty is sufficient

File: docs/UserGuide.md [hunk 3]
```diff
@@ -154,6 +155,7 @@ Format: `edit INDEX [r/ROLE] [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* The result of an edit can be identical to the original person, but it **cannot be identical to any other existing user in the address book**.
```
Comment (seah-minlong) [inline docs/UserGuide.md:158]: I don't think there is a need to specify this line? I don't think its considered a bug when you can edit the person to itself. It doesn't seem to add much value then

**Review by seah-minlong — CHANGES_REQUESTED** [review]
Some comments

File: docs/UserGuide.md (hunk 2 above)
Comment (zzzlou) [reply docs/UserGuide.md]: ok changes pushed

File: docs/UserGuide.md (hunk 3 above)
Comment (zzzlou) [reply docs/UserGuide.md:158]: I think there is no harm in mentioning it as it clarifies the intended behavior of the software.

**Review by seah-minlong — APPROVED** [review]
LGTM

**Pull request merged**